.github/workflows/ci.yml

```diff
@@ -3,7 +3,7 @@
 # SPDX-License-Identifier: MIT
 
 name: CI
-on: [push]
+on: [push, pull_request]
```

Comment:
Is this really necessary? My understanding is that this will result in the workflow running twice - first when the branch is pushed, and a second redundant time when the PR is created.
Is the point here to make sure the github PR captures the CI as "Checks" when the PR source branch is in someone else's fork of the repo?

Reply:
With only push, the workflow will run on your fork when you push, but when you 
submit a pull request the workflow will not run. It is true that the workflow
will run twice when merged, but I think it worth it and can help contributors
to detect issues.

I use the same setting in other projects, for example:
https://github.com/oVirt/ovirt-imageio/pull/21

On the other hand I'm using github actions only few weeks, so maybe I'm not
using it in best way :-)

Reply:
I see what you mean. I'd tend to think that having the workflow run in the forked repo is generally enough for the reviewer to verify before merging that the changes pass the tests.   
It's a bit unfortunate that Github does not pick those "remote" workflows as being part of the PR's checks. I can also see how that does allow to bypass the "checks" to merge changes that don't pass the tests, and puts some "very automatable" responsibility on the reviewer.

Ideally it'd be great to avoid running duplicate workflow (which I believe will happen upon every push event tracked by a PR), but this is no big deal at all.  
It's literally the first time I use Github Actions so you're the more experienced one here! I don't have a solution to offer although I've admittedly not spent any time digging into it.

Reply:
@cdessez thanks for reviewing!